docs/development/plugins/functionality/index.md

```diff
@@ -173,3 +173,10 @@ The plugins can have user-configurable settings that can be used to change the b
 - Example plugin shows [How to create plugin settings and use them](https://github.com/kubernetes-sigs/headlamp/tree/main/plugins/examples/change-logo)
 
 ![screenshot of the plugin settings](../images/plugin-settings.png)
+
+### App Theme
+
+Add a new custom Headlamp theme using [registerAppTheme](../../api/plugin/registry/functions/registerapptheme).
+Theme definition is an object, name must be unique and you must specify 'light' or 'dark' base for the theme.
```

Comment:
I don't see in the example plugin where it specifies the base theme?

Reply:
added it to the example and updated this sentence because 'base' property is not actually mandatory